CodeStarterCamp_Week2/Week2Step2.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Week2Step2.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by mingmac on 2022/04/26.
+//
+
+import Foundation
+
+func createRandomLottoSet() -> Set<Int> {
```

Comment:
[API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/)의 Naming -> Promote Clear Usage
부분에 아래 두 항목을 보시면 
- Omit needless words.
- Name variables, parameters, and associated types according to their roles, rather than their type constraints.

네이밍시 타입을 포함하는 것보다는 함수가 하는 역할에 맞게 이름을 지으라는 내용이 있습니다. 따라서 타입인 Set이 포함되는 것 보다는 랜덤한 수들을 만드는 것이니 이에 맞게 네이밍을 변경해볼 수 있을 것 같아요

Reply:
수정하였습니다 :D